charts/jaeger/values.yaml

```diff
@@ -200,6 +200,9 @@ agent:
       # type: RollingUpdate
       # rollingUpdate:
       #   maxUnavailable: 1
+  # Set to false if your release is called `jaeger`.
+  # Read More: https://github.com/jaegertracing/jaeger/issues/1986
```

Comment:
```suggestion
```

The issue link has been worked around, and for prometheus there are other workarounds such as adding `JAEGER_AGENT_PORT` in prometheus server container's env.

That aside adding the enableServiceLinks bool to the agent and any other components is a welcome addition!